docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -76,17 +63,9 @@ const admin = Role('http://example.com/login', async t => {
 
 After you create the roles, you can switch between users at any moment except for the role initialization code.
```

Comment:
Change "After you create the roles" to "After you create roles"

Also, can you clarify what you mean by "role initialization code"?